cortex.tf

```diff
@@ -0,0 +1,144 @@
+locals {
+  cortex = merge(
+    local.helm_defaults,
+    {
+      name                      = "cortex"
+      namespace                 = "cortex"
+      chart                     = "cortex"
+      repository                = "https://cortexproject.github.io/cortex-helm-chart"
+      service_account_name      = "cortex"
+      create_iam_resources_irsa = true
+      enabled                   = false
+      chart_version             = "0.4.0"
+      version                   = "v1.7.0"
+      iam_policy_override       = ""
+      default_network_policy    = true
+      cluster_name              = "cluster"
+    },
+    var.cortex
+  )
+
+  values_cortex = <<VALUES
```

Comment:
these values does not seems to be the values for cortex https://github.com/cortexproject/cortex-helm-chart/blob/master/values.yaml but the values from cluster autoscaler